Include/zipimport.h

```diff
@@ -0,0 +1,26 @@
+#ifndef Py_ZIPIMPORT_H
+#define Py_ZIPIMPORT_H
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* zipimporter object definition and support */
+
+struct _zipimporter {
+    PyObject_HEAD
+    PyObject *archive;  /* pathname of the Zip archive,
+                           decoded from the filesystem encoding */
+    PyObject *prefix;   /* file prefix: "a/sub/directory/",
+                           encoded to the filesystem encoding */
+    PyObject *files;    /* dict with file info {path: toc_entry} */
+};
+
+typedef struct _zipimporter ZipImporter;
```

Comment:
It seems that somehow when including this file to test this that the following error sometimes happens:

```
zipimport.h(19): error C2079: 'ZipImporter_Type' uses undefined struct '_typeobject'
```